src/components/alert/__spec__.js

```diff
@@ -40,7 +34,7 @@ describe('Alert', () => {
     });
 
     describe("on internal elements", () => {
-      let wrapper = shallow(<Alert open={ true } title='Test' subtitle='Test' showCloseIcon={ true } />);
+      let wrapper = mount(<Alert open={ true } title='Test' subtitle='Test' showCloseIcon={ true } />);
```

Comment:
I don't see any changes in the component that suggests that we now need to use mount. Why has this been changed?